src/app/backend/version/version.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	flag "github.com/spf13/pflag"
+)
+
+// VERSION of this binary
+var VERSION = "UNKNOWN"
```

Comment:
This can be moved to main `dashboard.go` and `ldflags` adjusted. Rest of the code can be removed.

Reply:
How about `Version` defined in the `src/app/backend/client/manager.go`?

Reply:
`client/manager` holds the code for creation of Kubernetes API client. It's not a place for a version of our binary. `dashboard.go` is our main file and is better suited for this.

Reply:
If `Version` defined in the `dashboard.go`, how to reference this `Version` in `src/app/backend/client/manager.go` ? Should I add a `userAgent` field for `clientManager`, just like 
```
type clientManager struct {
	// Autogenerated key on backend start used to secure requests from csrf attacks
	csrfKey string
	// Path to kubeconfig file. If both kubeConfigPath and apiserverHost are empty
	// inClusterConfig will be used
	kubeConfigPath string
	// Address of apiserver host in format 'protocol://address:port'
	apiserverHost string
	// Initialized on clientManager creation and used if kubeconfigPath and apiserverHost are
	// empty
	inClusterConfig *rest.Config
	// Responsible for decrypting tokens coming in request header. Used for authentication.
	tokenManager authApi.TokenManager
	// Kubernetes client created without providing auth info. It uses permissions granted to
	// service account used by dashboard or kubeconfig file if it was passed during dashboard init.
	insecureClient kubernetes.Interface
       // add user agent
       userAgent string
}
```
And rewrite the `NewClientManager` function, just like 
```
func NewClientManager(kubeConfigPath, apiserverHost, userAgent string) ClientManager {
	result := &clientManager{
		kubeConfigPath: kubeConfigPath,
		apiserverHost:  apiserverHost,
                userAgent: userAgent,
	}

	result.init()
	return result
}
```
So we can pass the `Version` defined in `dashboard.go` to the `src/app/backend/client/manager.go`.

Reply:
Ahh, ok there would be circular dependency if we define version in `dashboard.go` and import it there. I am ok with moving it to `client/manager.go` then.

Reply:
What's the next? Should I modify the code again?